.zenodo.json

```
@@ -0,0 +1,34 @@
+{
+  "creators": [
+    {
+      "affiliation": "810 Labs",
+      "name": "Arsenovic, Alex",
+      "orcid": "0000-0002-8599-5873"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Hadfield, Hugo",
+      "orcid": "0000-0003-4318-050X"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Wieser, Eric",
```

Note that we can write these as
```suggestion
      "name": "Eric Wieser",
```
and bibtex will behave exactly as before, but zenodo will show our names without a comma.